cache/cache.cpp

```diff
@@ -225,6 +225,38 @@ size_t cache_manager::idx(const unsigned char *id) {
 
 using namespace ioremap::cache;
 
+static int n2_send_file_info_without_fd(struct dnet_net_state *st,
```

Comment:
This function doesn't send file info, it does only fill the response.

Reply:
fixed